src/example/example.service.ts

```diff
@@ -0,0 +1,93 @@
+export class ExampleService {
+  public calculateAverage(numbers: number[]): number {
+    if (!numbers.length) return 0;
+    let sum = 0;
+    for (const num of numbers) {
+      sum += num;
+    }
+    return sum / numbers.length;
```

Comment:
PullProofAI

Consider handling the case where `numbers.length` is 0 in the `calculateAverage` method to avoid division by zero.
```diff
+     if (numbers.length === 0) return 0;
```



<!-- This is an auto-generated comment by PullProofAI -->